scripts/gn/README.md

```diff
@@ -0,0 +1,44 @@
+### Ginan Python CLI Tools
+
+This is a collection of tools to help you prepare your own data to run with Ginan.
+
+## Getting started
+
+Ensure you have created a virtual environment in the `scripts` folder and installed the `requirements.txt`.
+
+Also ensure you have run `python3 scripts/s3_filehandler.py -d -p` to populate the `inputData` folder with the constants that `pea` depends on.
```

Comment:
I would like to replace this with the `auto_download_PPP` script. I think I've got most (all?) of the relevant inputs now covered, especially following the new ver 3 ppp_example template. 
I think for now this is ok, but we'll need to do some testing seeing how that script can be incorporated here

Reply:
Awesome, I'll take a look at `auto_download_PPP` and we can look at incorporating it in near future

Reply:
Actually, I realise now that it hasn't entered the GitHub repo yet, so maybe I'll send you a copy of that now and you can have a look at it from that version